src/app/features/collection/components/rarity-slider/rarity-slider.component.html

```diff
@@ -0,0 +1,14 @@
+<div class="tw-flex tw-flex-nowrap tw-overflow-x-auto">
+  <ion-card
+    *ngFor="let rarity of rarities()"
+    class="tw-flex-shrink-0 tw-flex-grow-0 tw-flex-auto tw-my-4 tw-mx-2"
+    [button]="true">
+    <ion-card-content
+      [ngClass]="{ active: currentSelectedRarity() === rarity }"

```

Comment:
can be simplified to [class.selected-rarity]="currentSelectedRarity() === rarity".
It is also nice to give a bit context in calss name, to easy find it via global search.